### ISSUE TITLE
"impression_count" is missing

### ISSUE THREAD

**Issue by sor4chi** [twitterdev/twitter-api-typescript-sdk#61]
Twitter has recently started publishing impression counts, so the API can now also take `impression_count` in `public_metrics`. On the other hand, you can no longer get `impression_count` from `non_public_metrics`.

### Expected behavior

CAN to get `impression_counts` from `public_metrics`
CANNOT to get `impression_counts` from `non_public_metrics`

### Actual behavior

CANNOT to get `impression_counts` from `public_metrics`
CAN to get `impression_counts` from `non_public_metrics`

### Steps to reproduce the behavior

please use this
https://github.com/sor4chi/repro-twitter-sdk-missing-params

**Comment by sor4chi** [comment]
Sorry, it looks like you just haven't generated the latest one.
https://github.com/twitterdev/twitter-api-typescript-sdk/pull/51
Merging and generating this would solve the problem.

**Comment by sboy99** [comment]
Has it been fixed

**Comment by sboy99** [comment]
![Screenshot from 2023-07-05 11-37-30](https://github.com/twitterdev/twitter-api-typescript-sdk/assets/86586277/836e39d4-e604-415b-9796-e9bf0575e86d)
![Screenshot from 2023-07-05 11-36-22](https://github.com/twitterdev/twitter-api-typescript-sdk/assets/86586277/4a5b46d2-6bf4-4e1e-87fe-44fb24f7ac43)

**Comment by sor4chi** [comment]
@sboy99 
I don't think it has been fixed yet. The repository itself doesn't seem to be maintained in the first place. Probably since Elon fired some employees.

**Comment by sor4chi** [comment]
If you want to use it, you can fork it yourself, import #51, and publish it to your npm scope.
